VideoWeb/VideoWeb/ClientApp/src/app/testing/mocks/data/conference-test-data.ts

```diff
@@ -324,7 +326,113 @@ export class ConferenceTestData {
 
     getListOfParticipantDetails(): ParticipantResponseVho[] {
         const participants: ParticipantResponseVho[] = [];
+
```

Comment:
This looks like a copy and paste of the previous method but for `ParticipantResponseVho`. And the following method returns a list of `ParticipantResponseVho`. Could you explain this change? I'm not able to follow

Reply:
This was due to the fact I wanted to set the linked_participants, and I have a flag hasInterpreterLink on the ParticipantResponse(VHO) class that needs to get set. Plus I was trying to limit the impact of changing test data as getListOfParticipantDetails seems to be used in the conference setup